pbf/api.py

```diff
@@ -73,18 +74,27 @@ def get_ip(request):
     else:
         return request.META["REMOTE_ADDR"]
 
+class ApiKey(APIKeyHeader):
+    param_name = "X-API-Key"
```

Comment:
Can't see this `param_name` mentioned elsewhere in this class. Ah, but `APIKeyHeader` that it's extending must be looking for it? Yes, that's what https://django-ninja.dev/guides/authentication/ says. Okay, no further questions then.

Reply:
This also threw me off first round I looked at it but:
```Note: param_name is the name of the GET parameter that will be checked for. If not set, the default of "key" will be used.```